### DISCUSSION TITLE
What does "load remote changes" do?

### DISCUSSION THREAD

**Discussion by jtherrmann** [Zettlr/Zettlr#4488] [Q&A] [answered]
Under https://docs.zettlr.com/en/reference/settings/#general, there's the option:

> **Always load remote changes to the current file:** If checked, Zettlr will silently reload any remote changes to your files. If unchecked, you will be asked for each file.

My understanding was that Zettlr does not offer cloud syncing. So, what does "load remote changes" mean in this context?

**Comment by nathanlesage** [comment, marked as answer]
There are basically two things that can change a file. The first one is obviously Zettlr itself. If you save a file in Zettlr, it knows that it has modified the file and thus properly reloads any changes (for example number of words, potential frontmatter title, etc).

The second way is for another program on the computer to do so. On the one hand, you can modify those files with other editor apps. On the other, cloud services can change files, if you have changed them on another device and now they have downloaded the most recent version.

The latter type of change are, from the perspective of Zettlr, remote, because the app didn't do it.

When you open a file in Zettlr, you have two copies of it on your computer: the one is the file on your hard drive, the other, however, is a copy in system memory (that's how Zettlr displays the file to you). When you now modify the file, you first modify only the memory copy. As soon as you save, this will write whatever is in memory to the file on your computer.

When you now modify a file in Zettlr but don't save it yet, and in the same moment, your cloud services changes the file as well, then you have the problem: which version do you want to keep? Zettlr can give you the choice to either keep what you see in Zettlr, or overwrite that with however the file now looks on your computer.

If you activate the setting to always load remote changes, Zettlr won't ask you and just silently overwrite what you see in the editor. For the upcoming 3.0 this is the new default because that is what makes most sense for people.

**Reply by jtherrmann** [reply]
I see, thanks for the thorough explanation! I think that the word "remote" is slightly misleading, as that usually refers to another computer (i.e. a server). Perhaps the word "external" would be slightly more clear.
